tests/resources/main.tf

```diff
@@ -2,11 +2,16 @@
 # SLZ OCP
 ##############################################################################
 
+locals {
+  override_json_path = "${path.module}/override-json-file/override.json"
```

Comment:
It's still failing even after updating override_json_path to "${path.module}/../../override-json-file/override.json".
 Error: Invalid function argument on .terraform/modules/landing_zone/patterns/roks/module/config.tf line 50, in locals:
At this line of code -->
https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/b53a094273ec630951d7e74919dd7c15b12591ec/patterns/roks/module/config.tf#L50

Reply:
I think it should be "${path.root}/../../override-json-file/override.json" instead of "${path.module}/../../override-json-file/override.json"
Will try this..